pwiz_tools/Skyline/Model/SmallMoleculeTransitionListReader.cs

```diff
@@ -1452,6 +1398,12 @@ public static string GetAcceptedIonMobilityUnitsString()
                     errMessage =
                         string.Format(Resources.SmallMoleculeTransitionListReader_ReadPrecursorOrProductColumns_Cannot_derive_charge_from_adduct_description___0____Use_the_corresponding_Charge_column_to_set_this_explicitly__or_change_the_adduct_description_as_needed_, adduct.AdductFormula);
                 }
+                else if (countValues < 2)
+                {
+                    errMessage = string.Format(getPrecursorColumns
+                            ? Resources.PasteDlg_ValidateEntry_Error_on_line__0___Precursor_needs_values_for_any_two_of__Formula__m_z_or_Charge_
+                            : Resources.PasteDlg_ValidateEntry_Error_on_line__0___Product_needs_values_for_any_two_of__Formula__m_z_or_Charge_, row.Index + 1);

```

Comment:
You might want to show a different error message here if the "Product Neutral Loss" column is displayed: in that case, they do not need "Formula".

Reply:
I think you can make the case the product neutral loss is a variant of formula. The message doesn't use any formal column names, so the ambiguity works in our favor.